app/test/test_datastore.py

```diff
@@ -0,0 +1,36 @@
+import unittest
```

Comment:
Seems this file was added by accident. 
Please remove this file as this has been tracked in https://github.com/aws-samples/medical-transcription-analysis/pull/23

Reply:
Done.